java/lib/NetworkTablesJNI.cpp

```diff
@@ -917,6 +917,90 @@ JNIEXPORT jobjectArray JNICALL Java_edu_wpi_first_wpilibj_networktables_NetworkT
 
 /*
  * Class:     edu_wpi_first_wpilibj_networktables_NetworkTablesJNI
+ * Method:    setDefaultBoolean
+ * Signature: (Ljava/lang/String;Z)Z
+ */
+JNIEXPORT jboolean JNICALL Java_edu_wpi_first_wpilibj_networktables_NetworkTablesJNI_setDefaultBoolean__Ljava_lang_String_2Z
```

Comment:
Because I wrote these when I didn't know how JNI worked well and thought I needed them. Fixed.